src/web/client/extension.ts

```diff
@@ -268,12 +269,108 @@ export function processWillSaveDocument(context: vscode.ExtensionContext) {
     );
 }
 
-export function processWillStartCollaboartion() {
+export function processWillStartCollaboartion(context: vscode.ExtensionContext) {
     if (isCoPresenceEnabled()) {
-        // TODO: Add copresence logic
+        processOpenActiveTextEditor(context);
+        createWebWorkerInstance(context);
     }
 }
 
+let worker: Worker;
+
+export function processOpenActiveTextEditor(context: vscode.ExtensionContext) {
+    try {
+        context.subscriptions.push(
+            vscode.window.onDidChangeTextEditorSelection(async () => {
```

Comment:
lets use onTabChange instead

Reply:
@tyaginidhi can you have a look at the changes made?

Reply:
correct - refer function "processWorkspaceStateChanges" in extension.ts
https://github.com/microsoft/powerplatform-vscode/blob/main/src/web/client/extension.ts#L257